.github/workflows/Publish to Docker Hub.yml

```diff
@@ -57,9 +57,12 @@ jobs:
             { name: "ods-api-db-ods-sandbox", path: "alpine/pgsql" },
             { name: "ods-api-swaggerui", path: "alpine" },
             { name: "ods-api-web-api", path: "alpine/pgsql" },
-            { name: "ods-api-web-api", path: "alpine/mssql" },            
+            { name: "ods-api-web-api", path: "alpine/mssql" },
```

Comment:
Unrelated to this story but this looks odd. Since we define the same name twice, are we pushing both pg and mssql versions to the same image name, ultimately overriding each other?

Reply:
we now publish the -mssql sufix as part of the tag instead of the image name, look [here](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/0ef79394cc1d785e78a7e4f74fd482a021e8b4be/Docker/Invoke-Build.ps1#L238)